aim/sdk/plotly_sequence.py

```diff
@@ -0,0 +1,38 @@
+from typing import Union, Tuple
+
+from aim.sdk.sequence import Sequence
+from aim.sdk.objects.plotly import Plotly
+
+
+class Plotlys(Sequence):
+    """Class representing series of Plotly objects or Plotly lists."""
+
+    @classmethod
+    def allowed_dtypes(cls) -> Union[str, Tuple[str, ...]]:
+        plotly_typename = Plotly.get_typename()
+        return plotly_typename, f'list({plotly_typename})'
```

Comment:
@gorarakelyan @mahnerak should we support lists of plotly figures as an input to `Run.track`?

Reply:
I do not think so. If user wants to track several plotly figures it will be better to log them in separate sequences imo.
In case of audios/videos list of values is supported because usually users track batches of samples not a single sample.
@alberttorosyan @devfox-se ^

Reply:
please remove `f'list({plotly_typename})'` from supported types based on @gorarakelyan's feedback.